doc/source/building/index.rst

```diff
@@ -220,6 +220,60 @@ your system.
         try again. The Fortran compiler should be installed as described in
         this section.
 
+  .. tab-item:: Windows on ARM64
+    :sync: Windows on ARM64
+
+    In Windows on ARM64, the set of a compiler options that are available for
+    building NumPy are limited. Compilers such as GCC and GFortran are not yet
+    supported for Windows on ARM64. Currently, the NumPy build for Windows on ARM64
+    is supported with MSVC toolchain only. The use of a Fortran compiler is more
+    tricky than on other platforms, because MSVC does not support Fortran, and
+    gfortran and MSVC can't be used together. If you don't need to run the ``f2py``
+    tests, simply using MSVC is easiest. Otherwise, you will need the following
+    set of compilers:
+
+    1. MSVC + Flang-new (``flang-new``)
```

Comment:
What about the `ifx` compiler and `lfortran`? Perhaps @certik knows better about ARM64 support (don't have a machine to test).

Otherwise this looks good.

Reply:
Note that in LLVM 20 (release imminent I think, it's at 20.1.0rc3 now), `flang-new` becomes simply `flang`. That would be good to include here.